app/src/main/java/org/greenstand/android/TreeTracker/capture/TreeCaptureViewModel.kt

```diff
@@ -30,8 +31,7 @@ class TreeCaptureViewModel(
 
     suspend fun captureLocation() {
         _state.value = _state.value?.copy(isGettingLocation = true)
-        treeCapturer.pinLocation()
-        _state.value = _state.value?.copy(isGettingLocation = false)
+        _state.value = _state.value?.copy(isLocationAvailable = treeCapturer.pinLocation(), isGettingLocation = false)
```

Comment:
This breaks the UI. 

Before: Clicking capture button shows loading spinner
Now: Clicking capture button shows nothing